Views/Shared/_LootCardPartial.cshtml

```diff
@@ -15,25 +15,48 @@
                 }
                 </p>
             </li>
-            <li class="list-group-item">
-                Existing requests:
-                @if (Model.AllRequests == null || Model.AllRequests.Count == 0)
-                {
-                    <i>None</i>
-                } else {
-                    <ul>
-                    @foreach (var playerPosition in Model.AllRequests.Keys)
+            <! –– Requests only show for items without owners ––>
+            <! –– It meanst that once you assign an item to someone, they won't show ––>
+            <! –– If that's a problem, remove this condition ––>
+            @if (!Model.Loot.HolderId.HasValue) 
+            {
+                <li class="list-group-item">
+                    Existing requests:
+                    @if (Model.AllRequests == null || Model.AllRequests.Count == 0)
                     {
-                        <li>
-                            (@playerPosition) @Model.AllRequests[playerPosition]
-                        @if (Model.SpecialRequests.ContainsKey(playerPosition)) {
-                            @:&nbsp;<i class="fas fa-exclamation-circle text-primary" title="@Model.SpecialRequests[playerPosition]"></i>
+                        <i>None</i>
+                    } else {
+                        <ul>
+                        @foreach (var playerPosition in Model.AllRequests.Keys)
+                        {
+                            <li>
+                                (@playerPosition) @Model.AllRequests[playerPosition]
+                            @if (Model.SpecialRequests.ContainsKey(playerPosition)) {
+                                @:&nbsp;<i class="fas fa-exclamation-circle text-primary" title="@Model.SpecialRequests[playerPosition]"></i>
+                            }
+                            </li>
                         }
-                        </li>
+                        </ul>
                     }
-                    </ul>
-                }
+                </li>
+            }
+            <li class="list-group-item">
+                <label for="select_loot_item_status_@(Model.Loot.Id)">Status: @(Model.Loot.Status)</label>
             </li>
+            <! –– If you can somehow grab a player by ID, that loop inside can be replaced ––>
```

Comment:
You can, via LINQ. `var player = Model.Players.SingleOrDefault(p => p.Id == player.Id);`. However, in this case it's not needed at all, since Loot contains a navigation property to the Player object at Model.Loot.Holder and you can reference it directly. Thus, the full code for this part becomes:

```
            @if (Model.Loot.HolderId.HasValue)
            {
                        <li class="list-group-item">
                            <label for="select_loot_item_holder_@(Model.Loot.Id)">Holder: @(Model.Loot.Holder.Name)</label>
                        </li> 
             }
```

without any loops and conditions, except for the check if the item has a holder.